tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -627,85 +627,85 @@
     <string name="about_license_busybox" translatable="false">Meefik/busybox license</string>
     <string name="about_site" translatable="false"><a href="https://invizible.net">https://invizible.net</a></string>
 
-    <string name="update_notification">Downloading file</string>
-    <string name="update_interrupt_notification">Download was stopped</string>
-    <string name="cancel_download">CANCEL DOWNLOAD</string>
+    <string name="update_notification">Downloading file…</string>
+    <string name="update_interrupt_notification">Downloading stopped…</string>
+    <string name="cancel_download">Cancel download</string>
     <string name="update_new_version">New Version:</string>
-    <string name="update_ipro_has_apdate">InviZible Pro update is available. Do you want to download and update it? The update will continue in the background.</string>
-    <string name="update_dnscrypt_has_apdate">DNSCrypt update is available. Do you want to download and update it? The update will continue in the background.</string>
-    <string name="update_tor_has_apdate">Tor update is available. Do you want to download and update it? The update will continue in the background.</string>
-    <string name="update_itpd_has_apdate">Purple I2P update is available. Do you want to download and update it? The update will continue in the background.</string>
-    <string name="update_checking_title">Checking Updates</string>
-    <string name="update_checking_message">Please wait while Updates are checking.</string>
+    <string name="update_ipro_has_apdate">Download and install the new version of InviZible Pro in the background?</string>
+    <string name="update_dnscrypt_has_apdate">Download and install the new version of DNSCrypt in the background?</string>
+    <string name="update_tor_has_apdate">Download and install the new version of Tor in the background?</string>
+    <string name="update_itpd_has_apdate">Download and install the new version of Purple I2P in the background?</string>
+    <string name="update_checking_title">Checking for New Versions…</string>
+    <string name="update_checking_message">Please wait while looking for new versions.</string>
     <string name="update_check_warning">The update server is temporarily unavailable. Please try again later.</string>
     <string name="update_check_warning_menu">The update server was unavailable.</string>
     <string name="update_fault">Update fault.</string>
-    <string name="update_check_no_update">No Updates were Found.</string>
+    <string name="update_check_no_update">Everything is up to date.</string>
```

Comment:
A nicer message, albeit possibly erroneous.
Is there otherwise an error message?

Reply:
Yep. That's better. You got it right.